hangman/game.py

```diff
@@ -1,25 +1,87 @@
 from .exceptions import *
+import random
 
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
-
-def _get_random_word(list_of_words):
-    pass
-
+""" Mask word"""
 
 def _mask_word(word):
-    pass
+  if word =='':
```

Comment:
you may use `if not word` to indicate "Falsey" for readability.. do check out the style guide written by Google Team - https://github.com/google/styleguide/blob/gh-pages/pyguide.md